### PR TITLE
chore(v3): reduce template zip files

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

File: templates/package.json [hunk 1]
```diff
@@ -46,31 +46,31 @@
...
+        "ts/sso-tab",
+        "ts/non-sso-tab",
+        "csharp/notification-http-timer-trigger",
+        "csharp/notification-http-trigger",
+        "csharp/notification-timer-trigger",
+        "csharp/notification-webapi",
+        "csharp/command-and-response",
+        "csharp/message-extension",
+        "csharp/sso-tab",
+        "csharp/non-sso-tab"
```
Comment (hund030) [inline templates/package.json:73]: This list is for sync up dependencies.

File: .github/scripts/template-zip-autogen-v3.sh [hunk 2]
```diff
@@ -14,15 +14,18 @@ TEMPLATE_OUTPUT_DIR=$1
 mkdir -p ${TEMPLATE_OUTPUT_DIR}
 
 TEMPLATE_LIST=$(jq -r '.templatesV3[]' ./templates/package.json)
```
Comment (yukun-dong) [inline .github/scripts/template-zip-autogen-v3.sh:16]: Seems TEMPLATE_LIST is not used in following script and it can be deleted from package.json?

Comment (hund030) [reply .github/scripts/template-zip-autogen-v3.sh:16]: Sure.

**Review by yukun-dong — APPROVED** [review]

File: packages/fx-core/src/component/generator/constant.ts [hunk 3]
```diff
@@ -13,3 +13,4 @@ export const templateBetaVersion = "templates-0.0.0-beta";
 export const sampleRepoName = "TeamsFx-Samples";
 export const errorSource = "GEN";
 export const componentName = "generator";
+export const commonTemplateName = "common";
```
Comment (yukun-dong) [inline packages/fx-core/src/component/generator/constant.ts:16]: Personal suggestion: This var can be renamed to like "generalTemplateGroupName".

**Review by yukun-dong — APPROVED** [review]

**Review by yukun-dong — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**